elodie/filesystem.py

```diff
@@ -508,10 +522,17 @@ def process_file(self, _file, destination, media, **kwargs):
         checksum_file = db.get_hash(checksum)
         if(allow_duplicate is False and checksum_file is not None):
             if(os.path.isfile(checksum_file)):
-                log.info('%s already exists at %s. Skipping...' % (
-                    _file,
-                    checksum_file
-                ))
+                if constants.delete_duplicates:
```

Comment:
Need to import constants at the top of the file.
https://travis-ci.org/jmathai/elodie/jobs/488449125